setup.cfg

```diff
@@ -7,8 +7,15 @@ per-file-ignores =
      src/ert/experiment_server/_server.py: E501
      # Ignore all protobuf v2 files
     *_pb2.py: E
-# We ignore only things that black takes (better) care of:
 ignore =
+    # whitespace before ':'; solved by black:
     E203
+    # line break before binary operator, solved by black:
     W503
+    B008
```

Comment:
Why did you choose those warnings to ignore?

Reply:
Also, how do you feel about adding comment of what we ignoring in the same way you did for E203 and W503? I know it will increase the file, but I think it will come handy for people reading this file later

Reply:
> Why did you choose those warnings to ignore?

It is the minimal set of ignores that makes `flake8` pass when `flake8-bugbear` is installed.

Reply:
> Also, how do you feel about adding comment of what we ignoring in the same way you did for E203 and W503? I know it will increase the file, but I think it will come handy for people reading this file later

I thought about it, and had it in in some version. Felt it become a little bit lengthy, but I can add it again.

Reply:
Maybe one line summary for all of this warnings? For example, exactly what you said above?
```
Minimal set of ignores that makes flake8 pass when flake8-bugbear is installed.

```

Reply:
Fixed!